### ISSUE TITLE
taking too much time when predicting object i have GPU Quadro P4000, with 8 gb ram why it's taking ?

### ISSUE THREAD

**Issue by DeveloperRachit** [fizyr/keras-retinanet#1336]
taking too much time when predicting object i have GPU Quadro P4000, with 8 gb ram why it's taking ?

**Comment by DeveloperRachit** [comment]
i am using pretrained model retinanet_resnet152_500_classes_0.4991.h5

**Comment by DeveloperRachit** [comment]
id nay information you want tell me i will give you

**Comment by DeveloperRachit** [comment]
boxes, scores, labels = model.predict_on_batch(np.expand_dims(image, axis=0))

that taking too much time

**Comment by DeveloperRachit** [comment]
model = models.load_model(model_path, backbone_name='resnet152')
this is my loading method of model

**Comment by DeveloperRachit** [comment]
that model having 500 classes

**Comment by DeveloperRachit** [comment]
its taking 40sec to detect a images object

**Comment by de-vri-es** [comment]
Please don't simply post new issues for closed issues. You can still continue the discussion in the closed issue.

The 40 seconds is an indication that your GPU is not being used. This is not a keras-retinanet specific issue, but a keras or tensorflow issue. Please make sure you have the correct GPU-enabled versions of both. Please refer to their documentation for the details on this.

**Comment by DeveloperRachit** [comment]
okay but memory is being used na in GPU

**Comment by DeveloperRachit** [comment]
how could say that gpu is not used
